tests/manual-test-cases/Group21-Registries/21-1-Whitelist.robot

```diff
@@ -0,0 +1,318 @@
+Suite Setup  Setup Harbor
+Suite Teardown  Harbor Test Cleanup
+
+*** Keywords ***
+Setup Harbor
+    Set Test Environment Variables
+
+    Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Pass Execution  Test skipped on ESXi as Harbor is only supported on VC
+    
+    # Install a Harbor server with HTTPS a Harbor server with HTTP
+    Install Harbor To Test Server  protocol=https  name=harbor-https
+    Set Environment Variable  HTTPS_HARBOR_IP  %{HARBOR-IP}
+
+    Install Harbor To Test Server  protocol=http  name=harbor-http
+    Set Environment Variable  HTTP_HARBOR_IP  %{HARBOR-IP}
+
+#    Set Environment Variable  HTTPS_HARBOR_IP  10.118.69.161
+#    Set Environment Variable  HTTP_HARBOR_IP  10.118.69.160
+    Get HTTPS Harbor Certificate
+
+Harbor Test Cleanup
+    ${out}=  Run  govc vm.destroy harbor-http
+    ${out}=  Run  govc vm.destroy harbor-https
+    Log To Console  Cleaning up Harbor servers
+
+Get HTTPS Harbor Certificate
+    [Arguments]  ${HARBOR_IP}=%{HTTPS_HARBOR_IP}
+    # Get the certificates from the HTTPS server
+    ${out}=  Run  wget --tries=10 --connect-timeout=10 --auth-no-challenge --no-check-certificate --user admin --password %{TEST_PASSWORD} https://${HARBOR_IP}/api/systeminfo/getcert
+    Log  ${out}
+    Move File  getcert  ./ca.crt
+
+Setup Test Image in Unified OVA
+    # Complete this once unified OVA scripts are done
+
+    [Arguments]  ${HTTP_HARBOR_IP}=%{HTTP_HARBOR_IP}  ${HTTPS_HARBOR_IP}=%{HTTPS_HARBOR_IP}
+    # Create a user on Harbor and add to the default public library
+#    Create A New User  test  test@ci.com  tester  Password1  none
+#    Add A User To A Project  test  library  developer
+
+    # Push a busybox image up to Harbor's public library
+#    ${rc}  ${output}=  Run And Return Rc And Output  docker login -u admin -p %{TEST_PASSWORD} %{HTTP_HARBOR_IP}
+#    Should Be Equal As Integers  ${rc}  0
+#    Log  ${output}
+#    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} login -u admin -p %{TEST_PASSWORD} %{HTTPS_HARBOR_IP}
+#    Should Be Equal As Integers  ${rc}  0
+#    Log  ${output}
+#    Should Contain  ${output}  Login Succeeded
+    ${rc}  ${output}=  Run And Return Rc And Output  docker pull busybox
+    Should Be Equal As Integers  ${rc}  0
+    Log  ${output}
+    ${rc}  ${output}=  Run And Return Rc And Output  docker tag busybox ${HTTPS_HARBOR_IP}/library/busybox
+    Should Be Equal As Integers  ${rc}  0
+    Log  ${output}
+    ${rc}  ${output}=  Run And Return Rc And Output  docker push ${HTTPS_HARBOR_IP}/library/busybox   
+    Should Be Equal As Integers  ${rc}  0
+    Log  ${output}
+#    ${rc}  ${output}=  Run And Return Rc And Output  docker tag busybox ${HTTP_HARBOR_IP}/library/busybox
+#    Should Be Equal As Integers  ${rc}  0
+#    Log  ${output}
+#    ${rc}  ${output}=  Run And Return Rc And Output  docker push ${HTTP_HARBOR_IP}/library/busybox   
+#    Should Be Equal As Integers  ${rc}  0
+#    Log  ${output}
```

Comment:
Not a big fan of random portions un-commented in the middle of an almost all commented out test.  It doesn't look like this is actually called anywhere either.  Should we just remove this whole keyword for now?